src/app/payment-hub/paymenthub.component.html

```diff
@@ -68,13 +55,18 @@ <h4 matLine>Export Incoming Request to Pay</h4>
             </mat-icon>
             <h4 matLine>Export Outgoing Request to Pay</h4>
           </mat-list-item>
-
         </mat-nav-list>
-
       </div>
-
+      <div fxFlex="50%">
+        <mat-nav-list>
+          <mat-list-item [routerLink]="['bulkbatchexport']">
+            <mat-icon matListIcon>
+              <fa-icon icon="download" size="sm"></fa-icon>
```

Comment:
These should not have been part of this PR.

Reply:
Resolved